extensions/amazon-aws/src/components/searchbar/aws-profile-dropdown.tsx

```diff
@@ -140,8 +148,19 @@ const useProfileOptions = (): ProfileOption[] => {
       configFile[name]?.region ||
       credentialsFile[name]?.region ||
       (includeProfile && configFile[includeProfile]?.region);
-
-    return { ...config, region, name };
+    const sso_start_url =
+      configFile[name]?.sso_start_url ||
+      credentialsFile[name]?.sso_start_url ||
+      (includeProfile && configFile[includeProfile]?.sso_start_url);
+    const sso_account_id =
+      configFile[name]?.sso_account_id ||
+      credentialsFile[name]?.sso_account_id ||
```

Comment:
From the official [docs](https://docs.aws.amazon.com/sdkref/latest/guide/file-format.html#file-format-creds):

> Only the following settings that are considered "secrets" or sensitive can be stored in the credentials file: aws_access_key_id,aws_secret_access_key, and aws_session_token. Although these settings can alternatively be placed in the shared config file, we recommend that you keep these sensitive values in the separate credentials file.

So it is redundant to check credentials file for region, start_url, account_id etc. and can be removed.

Reply:
ty! you can resolve this one as well